src/chocolatey.resources/helpers/functions/Install-ChocolateyZipPackage.ps1

```diff
@@ -71,6 +71,9 @@ likely your script folder. If unzipping to your package folder, the path
 will be like
 `"$(Split-Path -Parent $MyInvocation.MyCommand.Definition)\\file.exe"`
 
+.PARAMETER SpecificFolder
+OPTIONAL - This is a specific directory within zip file to extract.
+

```

Comment:
Thanks for putting that in!

Reply:
We may want to explain what version this is first available in or when it was fixed to work appropriately (the released version of choco where this was known to work) - it worked in earlier code, stopped working for awhile, and then started working again more recently when something was fixed.

Reply:
https://github.com/chocolatey/choco/issues/676 came out in 0.10.4

Reply:
I see you are simply bringing over https://github.com/chocolatey/choco/blob/0c720d6f7637acabe5341bff9fc94ced0bc6328e/src/chocolatey.resources/helpers/functions/Get-ChocolateyUnzip.ps1#L70-L71 to here. That's fine. We can leave out the details if you want.

Reply:
This was the specific commit that fixed this issue - https://github.com/chocolatey/choco/commit/3b13869a0819824d8a79e5a98e3d17386be62b3b